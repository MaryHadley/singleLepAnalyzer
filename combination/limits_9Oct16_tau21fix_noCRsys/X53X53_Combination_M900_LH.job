universe = vanilla
Executable = X53X53_Combination_M900_LH.sh
Should_Transfer_Files = YES
WhenToTransferOutput = ON_EXIT
Notification = Error
request_memory = 3072
Output = X53X53_Combination_M900_LH.out
Error = X53X53_Combination_M900_LH.err
Log = X53X53_Combination_M900_LH.log
Queue 1