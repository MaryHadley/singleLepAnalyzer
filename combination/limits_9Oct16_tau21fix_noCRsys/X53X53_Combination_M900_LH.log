000 (1297376.000.000) 10/10 19:27:08 Job submitted from host: <192.168.63.249:41533>
...
001 (1297376.000.000) 10/10 19:27:18 Job executing on host: <192.168.63.243:49490>
...
006 (1297376.000.000) 10/10 19:27:27 Image size of job updated: 1
	3  -  MemoryUsage of job (MB)
	2268  -  ResidentSetSize of job (KB)
...
006 (1297376.000.000) 10/10 19:32:28 Image size of job updated: 283528
	162  -  MemoryUsage of job (MB)
	165732  -  ResidentSetSize of job (KB)
...
010 (1297376.000.000) 10/10 19:55:02 Job was suspended.
	Number of processes actually suspended: 3
...
011 (1297376.000.000) 10/10 19:55:22 Job was unsuspended.
...
006 (1297376.000.000) 10/10 20:12:32 Image size of job updated: 284352
	163  -  MemoryUsage of job (MB)
	166648  -  ResidentSetSize of job (KB)
...
005 (1297376.000.000) 10/10 20:16:08 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:47:57, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:47:57, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1629  -  Run Bytes Sent By Job
	322  -  Run Bytes Received By Job
	1629  -  Total Bytes Sent By Job
	322  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :     1100        1   3824816
	   Memory (MB)          :      163     3072      3072
...
