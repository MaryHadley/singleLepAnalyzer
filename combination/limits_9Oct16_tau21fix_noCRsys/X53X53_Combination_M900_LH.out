/var/lib/condor/execute/dir_31646
Total number of likelihood evaluations: 55000100
Total number of likelihood evaluations: 5500100
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M900_LH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M900_LH/quant-toys:0-sig-76cb224ca2.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M900_LH/quant-data-sig-4b2c25a73f.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M900_LH
