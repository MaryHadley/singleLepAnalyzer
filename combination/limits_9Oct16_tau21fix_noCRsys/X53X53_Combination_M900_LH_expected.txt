# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.095529   0.048389    0.20002  0.066084    0.14073
